modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-oadp-rosa-sts_{context}"]
+= OADP on ROSA with STS
+
+THIS IS A DRAFT. It includes questions for Dev and QE in brackets. This draft is based on link:https://hackmd.io/yiZBeGFkSJOGIwni9iNnsA[Documentation for OADP on ROSA with STS <draft>].
+
+
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. A ROSA deployment of OpenShift is configured specifically for AWS services. AWS STS is a global web service that provides short-term credentials for IAM or federated users. ROSA with STS is the recommended credential mode for ROSA clusters.
+
+[The following sentence was in the source. Is it OK? is it too vague?] Not all of the features and options provided by OADP are recommended or supported while using ROSA clusters with OADP.
+
+[IMPORTANT]
+====
+Disable Restic for OADP in ROSA environments because Restic is not supported in such environments. Use CSI storage instead.
```

Comment:
```suggestion
Disable Restic for OADP in ROSA environments because Restic is not supported in such environments. Use Native snapshots workflow instead.
```